Autopilot update + formatting fixes (#90)
Capstone-Skynet/Capstone-Skynet.github.io: docs/design/3_2_multirotor_v2.tex

\end{figure}

\subsubsection{Flight Computer and Sensors}

> To allow for a user with minimal flight training to use the multicopter successfully, a standalone flight controller is integrated into the multicopter system to allow for the automation of a majority of flight operations. The flight controller unit (FCU) are also refered to as ``autopilot'' for their autonomous abilities. 

Reviewer: ```suggestion
To allow for a user with minimal flight training to use the multicopter successfully, a standalone flight controller unit is integrated into the multicopter system to allow for the automation of a majority of flight operations. The flight controller unit (FCU) is also often referred to as ``autopilot'' for its autonomous abilities. 
```

	\label{flightdisplay}
\end{figure}

\paragraph{Flight Computer Unit and Software}

> The flight controller included on the multirotor is called ArduPilotMega (APM) 2.8 using firmware of the same name. APM is a ubiquitousopen-source hardware and firmware stack. APM Planner 2.0 is a ``ground station'' software package used to configure the autopilot before flight operations such as installing firmware, or during flight operations to provide real time telemetry \footnote{Telemetry requires additional hardware that is currently not supported on the multirotor}. As seen in Figure \ref{flightdisplay}, APM Planner 2.0 provides users with easy-to-use GUI for tuning multirotor settings and paramters. The choice of flight controller and its dedicated software was necessitated by cost-limitations at purchasing time, however the differences between budget flight controllers is ultimately marginal.

Reviewer: ```suggestion
The flight controller included on the multirotor is called ArduPilotMega (APM) 2.8 using firmware of the same name. APM is a ubiquitous open-source hardware and firmware stack. APM Planner 2.0 is a ``ground station'' software package used to configure the autopilot before flight operations such as installing firmware, or during flight operations to provide real time telemetry \footnote{Telemetry requires additional hardware that is currently not supported on the multirotor}. As seen in Figure \ref{flightdisplay}, APM Planner 2.0 provides users with easy-to-use GUI for tuning multirotor settings and parameters. The choice of flight controller and its dedicated software was necessitated by cost-limitations at purchasing time, however the differences between budget flight controllers is ultimately marginal.
```
